js/main.js

```diff
@@ -1,63 +1,71 @@
-let books = [];
+class BookCollection {
+  #index;
 
-function removeBook(id) {
-  const newArray = books.filter((book2) => book2.id !== id);
-  localStorage.setItem('books', JSON.stringify(newArray));
-}
+  books;
 
-function showBook(book) {
-  let element;
-  const div = document.createElement('div');
-  element = document.createElement('p');
-  element.appendChild(document.createTextNode(book.title));
-  div.appendChild(element);
-  element = document.createElement('p');
-  element.appendChild(document.createTextNode(book.author));
-  div.appendChild(element);
-  element = document.createElement('button');
-  element.setAttribute('type', 'button');
-  element.appendChild(document.createTextNode('Remove'));
-  element.addEventListener('click', () => {
-    // remove the book from the screen
-    div.remove(book.id);
-    // remove the book from the local storage
-    removeBook(book.id);
-  });
-  div.appendChild(element);
-  element = document.createElement('hr');
-  div.appendChild(element);
-  document.getElementById('book-cont').appendChild(div);
-}
+  Constructor(books = []) {
+    this.#index = 0;
+    this.books = books;
+  }
 
-function addBook() {
-  const randomId = Math.floor(Math.random() * 100);
-  const element = {
-    id: randomId,
-    title: document.getElementById('title').value,
-    author: document.getElementById('author').value,
-  };
-  books.push(element);
+  addBook(tit, auth) {
```

Comment:
- [ ] @elafildecolor and @chiztechnology great work on your project however I am unable to add books using your below video demonstration. 

https://user-images.githubusercontent.com/61408860/204641276-b5876d38-af0c-4b2a-8ee5-20e77420a4fa.mp4